loomi/regulariza-diagonal-client
# Feature/spouse details

## Feature Spouse Details

**What I did:**

- Add spouse inputs in DetailsResident.
- Create titles to toggle between inputs

**How to test:**

1. In your browser go to [stage](http://project.netlify.app/) or [localhost](http://localhost:3000/)
2. Check if the component/view looks just like the [mockup](https://xd.adobe.com/) in different screen sizes (width between 1024px and 1920px)
3. Test the things that you think are worth testing, even the ones that are not in this description

![image](https://user-images.githubusercontent.com/32523401/126669416-36989788-1f30-46ac-962f-1c7ceb14f998.png)
